index.html

```diff
@@ -2,8 +2,85 @@
 <html lang="en">
 <head>
     <title>James Cullen Portfolio</title>
+    <link rel="stylesheet" type="text/css" href= "styles.css"/>
 </head>
-<body>
+<body class="style">
```

Comment:
Could you use the 'body' tag as a selector? Also 'style' might cause issue's as may be confused with styles.css

Reply:
'Style' class omitted and relevant CSS functions put into body tag a a selector.